include/tsid/solvers/fwd.hpp

```diff
@@ -81,8 +83,8 @@ namespace tsid
     { return aligned_pair<T1,T2>(t1,t2); }
     
     
-    typedef pinocchio::container::aligned_vector< aligned_pair<double, math::ConstraintBase*> > ConstraintLevel;
-    typedef pinocchio::container::aligned_vector< aligned_pair<double, const math::ConstraintBase*> > ConstConstraintLevel;
+    typedef pinocchio::container::aligned_vector< aligned_pair<double, std::shared_ptr<math::ConstraintBase>> > ConstraintLevel;
+    typedef pinocchio::container::aligned_vector< aligned_pair<double, std::shared_ptr<const math::ConstraintBase>>> ConstConstraintLevel;
```

Comment:
the `>>` and `>>>` instead of `> >` and `> > >` will probably fail on older GCC version, even if we add the `-std=c++11` flag. Could you add the spaces ?